individual-files/ServerThread.java

```diff
@@ -0,0 +1,49 @@
+import java.io.BufferedReader;
```

Comment:
Extraneous (E) - lines 1 through 6 are redundant as lines 8 through 10 import the entire libraries.